### PR TITLE
Update localisations and crowdin configuration

### PR DESCRIPTION
**Pull request by peppy**
Probably need to to a vague check this time, it's been piling up (had some issues with the crowdin dumping configuration)

### CODE REVIEW

File: resources/lang/fi/store.php [hunk 1]
```diff
@@ -105,7 +105,7 @@
     'supporter_tag' => [
         'gift' => 'lahjoita pelaajalle',
         'require_login' => [
-            '_' => 'Ollaksesi Tukija sinun täytyy olla :link!',
+            '_' => '',
```
Comment (notbakaneko) [inline resources/lang/fi/store.php]: Some of the translations got removed like this one, but I can't seem to find any history of the change

Comment (TPGPL) [reply resources/lang/fi/store.php]: Yeah this one was available to translate on crowdin for no reason

Comment (nanaya) [reply resources/lang/fi/store.php]: original translation has been changed so looks like crowdin assumes it's a different thing.

Comment (TPGPL) [reply resources/lang/fi/store.php]: Oh right i think it was after every `supporter tag` was changed into `osu!supporter tag`

**Pull request merged**